RailwayCo/Assets/Scripts/Unity/Tooltip/Tooltip.cs

```diff
@@ -0,0 +1,64 @@
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+[ExecuteInEditMode()]
+public class Tooltip : MonoBehaviour
+{
+    public TextMeshProUGUI headerField;
+    public TextMeshProUGUI contentField;
+    [SerializeField] private LayoutElement layoutElement;
+    [SerializeField] private int characterWrapLimit;
+    private RectTransform _rectTransform;
+
+    private void Awake()
+    {
+        _rectTransform = GetComponent<RectTransform>();
+    }
+
+    private void Update()
+    {
+        if (Application.isEditor) 
+            CheckTextLength();
```

Comment:
Purpose of putting this in editor mode and over here?

Reply:
So that it rescales in editor mode to preview the content in the tooltip